EP-1146: Version 2 updates.

MitchellLawson-Burton opened the pull request:
## [2.0.0] - 2023-12-06

### Removed

- Removed the min-height set on the iframe element.

### Added

- `allow-downloads` permission to the iframe sandbox attribute. This allows users to download attachments from class notes.

MitchellLawson-Burton committed:
Remove the minHeight on the iframe.

MitchellLawson-Burton committed:
Version 2.0.0.

MitchellLawson-Burton committed:
Add prettier config.

prisetup reviewed (approved):
Looks good to me.

prisetup commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@getsetup/embed",
-  "version": "1.7.0",
+  "version": "2.0.0",
```
Are we considering this Major version update?

MitchellLawson-Burton replied:
Yeah. It probably won't affect anyone, but removing the `min-height` on the iframe is technically a breaking change. And since this package follows semver, we need to bump to version 2.

MitchellLawson-Burton committed:
Add allow-downloads to iframe sandbox permissions.

Send sandbox permissions to the iframe via the query string.
So the iframe can toggle features based on permissions.
